### PR TITLE
Sync various sections

### PR DESCRIPTION
**Pull request by saundefined**
Fix the build

### CODE REVIEW

File: language/oop5/basic.xml [hunk 1]
```diff
@@ -123,6 +123,73 @@ Stack trace:
...
     </example>
    </warning>
+
+    <sect3 xml:id="language.oop5.basic.class.readonly">
+      <title>Classes en lecture seule (readonly)</title>
+      <para>
+        À partir de PHP 8.2.0, une classe peut être marquée
+        avec le modificateur <modifier>readonly</modifier>.
+        Marquer une classe comme <modifier>readonly</modifier> ajoutera
+        le <link linkend="language.oop5.properties.readonly-properties"><modifier>readonly</modifier> modificateur</link>
```
Comment (devnexen) [inline language/oop5/basic.xml:133]: I would suggest
`le <link linkend="language.oop5.properties.readonly-properties">modificateur <modifier>readonly</modifier></link>`

**Comment by devnexen** [comment]
looks ok to me cc @Girgias

File: language/oop5/basic.xml [hunk 2]
```diff
@@ -123,6 +123,73 @@ Stack trace:
...
+// Fatal error: Cannot apply #[AllowDynamicProperties] to readonly class Foo
+?>
+]]>
+        </programlisting>
+      </example>
+
+      <para>
+        Comme ni les propriétés non typées, ni les propriétés statiques
+        ne peuvent être marquées avec le modificateur <literal>readonly</literal>,
+        les classes readonly ne peuvent pas non plus les déclarer :
```
Comment (Girgias) [inline language/oop5/basic.xml:157]:
```suggestion
        Comme ni les propriétés non typées, ni les propriétés statiques
        ne peuvent être marquées avec le modificateur <literal>readonly</literal>,
        les classes readonly ne peuvent pas non plus les déclarer :
```
Nit: The indentation seems to be wonky but that's not important

File: language/oop5/basic.xml [hunk 3]
```diff
@@ -123,6 +123,73 @@ Stack trace:
...
+}
+
+// Fatal error: Readonly class Foo cannot declare static properties
+?>
+]]>
+        </programlisting>
+      </example>
+      <para>
+        Une classe <modifier>readonly</modifier> peut être
+        <link linkend="language.oop5.basic.extends">extended</link> si,
```
Comment (Girgias) [inline language/oop5/basic.xml:187]:
```suggestion
        <link linkend="language.oop5.basic.extends">étendu</link> si,
```

**Review by Girgias — APPROVED** [review]
Thanks! :)

**Comment by devnexen** [comment]
Ah I thought the `extended` anchor was valid even in french, good to know.

**Pull request merged**